### ISSUE TITLE
[FEATURE] Add support for environment variables

### ISSUE THREAD

**Issue by menathan** [restechnica/semverbot#78]
Use case: For mono repo builds, dynamically set a semver `suffix` in the build pipeline.  

Suggested implementation:

```sh
export SBOT_GIT_TAGS_SUFFIX="-my-product"
sbot release version
```

.. would return something like `0.1.0-my-product`.

Suggestion is to implement 2 variables for now: 

- `SBOT_GIT_TAGS_PREFIX`
- `SBOT_GIT_TAGS_SUFFIX`

These variables would take precedence over the configuration set in the sbot config file.

**Comment by shiouen** [comment]
Fixed in #77

**Comment by shiouen** [comment]
Fixed in #77
